src/tools/asm2wasm.cpp

```diff
@@ -87,7 +87,7 @@ int main(int argc, const char *argv[]) {
            [&trapMode](Options *o, const std::string &argument) {
              try {
                trapMode = trapModeFromString(argument);
-             } catch (std::invalid_argument e) {
+             } catch (std::invalid_argument &e) {
```

Comment:
please put the `&` on the type

Reply:
Sure, I can do that.  But FWIW, nearby code has a lot of `const std::string &argument`, which is why I wrote it this way.

Reply:
Oh, oops, those are wrong too then ;) I'll fix those later.